spec/index.html

```diff
@@ -1146,7 +1146,17 @@ <h2>RDF/XML Internet Media Type, File Extension, and Macintosh File Type</h2>
 
   <section class="informative" id="privacy-considerations">
     <h2>Privacy Considerations</h2>
-    <p class="ednote">TODO</p>  
+    <p>RDF/XML is used to express arbitrary application data,
```

Comment:
As with the others...
```suggestion
    <p>The RDF/XML format is used to express arbitrary application data,
```